src/manifest_parser_test.cc

```diff
@@ -24,7 +24,7 @@
 struct ParserTest : public testing::Test,
                     public ManifestParser::FileReader {
   void AssertParse(const char* input) {
-    ManifestParser parser(&state, this);
+    ManifestParser parser(&state, this, false);
```

Comment:
Did you want me to send a followup for all the |false|s? It seemed a bit noisy to put it in every test.